### ISSUE TITLE
Add translations for ALFONSÓ languages

### ISSUE THREAD

**Issue by luksan47** [luksan47/mars#74]
[just for fun]
Add translation for

- [x] Latin
- [ ] French
- [x] Italian
- [ ] German
- [x] Spanish
- [ ] Ancient greek

For english continuous translation is required.

**Comment by Vilandry** [comment]
Gonna work on the french translation ^^

**Comment by luksan47** [comment]
I'll take latin and ancient greek.

**Comment by MattiaEffendi** [comment]
Actually working on the Italian translation.

**Comment by MattiaEffendi** [comment]
#84

**Comment by luksan47** [comment]
@kcscs will extend the selector

**Comment by luksan47** [comment]
@kcscs or @uno20001 are you working on the language selector or should I take over?

**Comment by uno20001** [comment]
Will do it next week if it's not urgent.

**Comment by luksan47** [comment]
Tomorrow or the day after would be the best.

**Comment by matimc** [comment]
Some remarks (mostly on the Latin language which must be the most important and essential feature in the new release):
– Is it on purpose that the title of the homepage is Laravel in every language?
– I believe that the sapien would be sapienS correctly
– On the printing page: the "cost" and "state" must be translated, because I had to use Google Translate for them...  (My candidates would be sumptus and status.)
– On the internet page: as I remember, aranea is only used for the spider's web, the modern term network is better approximated by reticulum (also reticulum in the correct form).
– Where is the "Nem baj" and the "Új jelszó generálása" in the language files? I couldn't find them but shouldn't they also be translated (for each language)?
– Other than that, I ran through the source and I think that there are some places where the order of words aren't exactly correct for the classical usage of the language: e.g. the name of the workshops should be Officina X.
– And there are several missing translations in the user.php.
Hope I'm not wrong with these comments and a big high five for you for both the idea and the implementation. Finally I can use Urán in my preferred language! ^^
P.S.: Next time the comments will also be in Latin!
